src/components/SearchView.tsx

```diff
@@ -1,172 +0,0 @@
-import * as React from "react";
```

Comment:
@fanwgwg [has changed something](https://github.com/DynamoDS/librarie.js/pull/42/files) in `SearchView` so please make sure those changes persist over here, too. @fanwgwg can you help review this when YX makes changes? Thanks!